documentation/src/main/asciidoc/topics/proc_configure_clients_trust_server.adoc

```diff
@@ -13,6 +13,7 @@ the `ConfigurationBuilder` or by specifying properties in
 . Enable SSL with the `ssl()` method. This implicitly enables SSL if you specify a truststore.
 . Specify the name of the truststore with the `trustStoreFileName()` method.
 . Specify the truststore secret with the `trustStorePassword()` method.
```

Comment:
I guess this isn't required with "PEM" trust stores right? (at least it works without it)